deps/src/coeffs.cpp

```diff
@@ -44,9 +44,8 @@ void singular_define_coeffs(jlcxx::Module & Singular)
         }
     });
 
-    Singular.method("n_Delete_Q", [](void * n, coeffs cf) {
-        number   tt = reinterpret_cast<number>(n);
-        number * t = &tt;
+    Singular.method("n_Delete_Q", [](snumber * n, coeffs cf) {;
+        number * t = &n;
         if (n != NULL) {
             n_Delete(t, cf);
```

Comment:
Why introduce that helper variable at all? You can just do this:
```suggestion
            n_Delete(&n, cf);
```

Reply:
Classic no-brainer actually. it is a leftover when there was still a cast from `void*`